skyportal/handlers/api/source.py

```diff
@@ -569,6 +572,15 @@ async def get_sources(
     See Source Handler for optional arguments
     """
 
+    if has_no_tns_name:
+        raise NotImplementedError(
+            "hasNoTNSName is not implemented here yet, use experimental handler with useExperimentalSources=True"
```

Comment:
useExperimental vs useExperimentalSources. I prefer the latter.

Reply:
Ah you are right! Thanks for the suggestion